### PR TITLE
feat: [CDB-1067]: enable dashboards for CCM team license

### PR DESCRIPTION
**Pull request by sean-harness**
### Summary

This is a small change to allow users to access dashboards when they have a CCM Team edition license on their account.
* Also some cleanup of old unused redirects.

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by sean-harness** [comment]
retrigger titlecheck

File: src/modules/45-dashboards/RouteDestinations.tsx [hunk 1]
```diff
@@ -93,7 +90,6 @@ export const CdbMfeRoutes = (
     <RouteWithLayout
       layout={MinimalLayout}
       path={[
-        routes.toOldCustomDashboard({ ...accountPathProps }),
```
Comment (Harness-JKhattak) [inline src/modules/45-dashboards/RouteDestinations.tsx:96]: Taking this out will result in there being no redirect in place when toggling between the old UI and new UI due to the difference in URL structure

Comment (sean-harness) [reply src/modules/45-dashboards/RouteDestinations.tsx:96]: I don't think this is related to the new new UI. This was in place for a transition from the CG UI to the NG UI.

**Comment by sean-harness** [comment]
retrigger cypress-pipeline

**Comment by sean-harness** [comment]
retrigger cypress-pipeline

File: src/modules/45-dashboards/RouteDestinations.tsx (hunk 1 above)
Comment (Harness-JKhattak) [reply src/modules/45-dashboards/RouteDestinations.tsx:96]: Here’s the FF to toggle on/off to test that re-routing out between the old UI to MFE -> https://uat.harness.io/ng/account/I3TdrVLBSd2Zr2ttor7Z2A/cf/orgs/default/projects/HarnessFeatureFlags/feature-flags/CDB_MFE_ENABLED?activeEnvironment=QA

**Comment by sean-harness** [comment]
retrigger cypress-pipeline

**Review by Harness-JKhattak — APPROVED** [review]

**Comment by sean-harness** [comment]
retrigger cypress-pipeline

**Review by abhinavrastogi-harness — APPROVED** [review]

**Pull request merged**